gef.py

```diff
@@ -3246,22 +3246,25 @@ class RemoteCommand(GenericCommand):
     def __init__(self):
         super(RemoteCommand, self).__init__(prefix=False)
         self.add_setting("proc_directory", "/proc")
+        gdb.events.cont.connect(self.continue_handler)
```

Comment:
Is there any particular reason you've used to add a GDB `continue` event handler instead of `gdb.events.new_objfile`? 
Not that I care so much about performance, but your handler will be executed **every** time GDB continues an execution. By the look of your patch, I believe `new_objfile` would be more appropriate, but maybe there is a reason you did not use it.